persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1487,15 +1493,24 @@ tableName = escapeE . tableDBName
 fieldName :: (PersistEntity record) => EntityField record typ -> Text
 fieldName = escapeF . fieldDBName
 
+fieldName' ::  forall record typ. (PersistEntity record, PersistEntityBackend record ~ SqlBackend) => EntityField record typ -> FieldNameDB
+fieldName' f = fieldDB $ persistFieldDef f
+
 escapeC :: ConstraintNameDB -> Text
 escapeC = escapeWith escape
 
 escapeE :: EntityNameDB -> Text
 escapeE = escapeWith escape
 
+escapeES :: EntityNameDB -> String
```

Comment:
same here; the implementation that I copied from `MySQL.hs` converted `EntityNameDB`s to `String`s instead of `Text`, so I updated the methods in this file to convert to `Strings` for my use case.  Should I just refactor the method to work with `Text` and then remove these (this, and then the `escapeFS` below) methods?